### ISSUE TITLE
Dark mode option independent of OS system

### ISSUE THREAD

**Issue by OneCreek** [babybuddy/babybuddy-for-android#80]
Would it be possible to have a setting - additionally to overtaking the system theme mode - to also set a manual overrule for light or dark mode?

**Comment by MrApplejuice** [comment]
Definitely possible, quite easy actually compared to the other stuff... I will plan it in for the next feature release.

**Comment by OneCreek** [comment]
How can it be the graphics look so different on a 

Google Pixel 
![Screenshot_20241004-221434.png](https://github.com/user-attachments/assets/6cde4405-d2f4-4b5e-9560-faa5b669a18b)

Compared to a 

Xiaomi 
![Screenshot_2024-10-04-22-09-37-710_eu.pkgsoftware.babybuddywidgets~2.jpg](https://github.com/user-attachments/assets/ea398b87-5323-480c-ba49-e4d27d14cd47)

**Comment by OneCreek** [comment]
> Hi @OneCreek!
> 
> I do not have a Xiaomi phone and cannot reproduce the error on my side. So I "talked it out" with ChatGPT instead... results inconclusive, though it maybe had some relevant points:
> 
> - Something that we could test together: There seems to be an option `android:forceDarkAllowed="false"` which might be relevant. It might prevent the Xiaomi Android version from overloading my app's dark mode theme. I could upload a "beta version" to google-play so that you can test it (you would need to opt in to that, and see note below)
> - ChatGPT pointed me to "Xiaomi devices have options for color profiles in display settings". Seems to check out according to this video: https://youtu.be/3Rgqo5aTB8c?t=44 Are any of these options active that are "not standard" on the Xiaomi phone, maybe? Those definitely might cause discoloring.
> 
> _For the first option, we still need to wait for a resolution to my appeal to Google regarding the suspected "sexual content" policy violation on  the Google play store. So this might take another week or so... cannot quite tell at the moment._
> 

Yes I think it was due to a Xiaomi individual override functionality! Thanks for the research and support.

Waiting for the app specific support:)
